src/test/java/io/micrometer/logzio/LogzioMeterRegistryTest.java

```diff
@@ -86,6 +80,62 @@ static void destroy() {
         registry.close();
         server.stop();
     }
+//    @Test
```

Comment:
why under comment?

Reply:
Still haven't created tests for the new features